src/commands/up.rs

```diff
@@ -66,10 +70,21 @@ pub struct UpCommand {
         name = BINDLE_PASSWORD,
         long = "bindle-password",
         env = BINDLE_PASSWORD,
-        requires = BINDLE_USERNAME
+        requires = BINDLE_USERNAME,
+        hide = true,
     )]
     pub bindle_password: Option<String>,
 
+    /// Reference to run the application from a container registry.
+    #[clap(
+        name = REFERENCE_OPT,
+        long = "reference",
+        env = REFERENCE_OPT,
+        conflicts_with = BINDLE_ID_OPT,
+        conflicts_with = APP_CONFIG_FILE_OPT
+    )]
+    pub reference: Option<String>,
+
     /// Ignore server certificate errors from bindle server
```

Comment:
Looks like this is also used for registries.

Reply:
@itowlson I'm not sure what this comment is suggesting.

Reply:
This ignores certificate errors from the registry as well.

Reply:
Ah. So just adding a `conflicts_with = INSECURE_OPT` is the ask?

Reply:
Nevermind, I see. Just a comment update.